stock_picking_report_custom_description/models/stock_rule.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockRule(models.Model):
+    _inherit = "stock.rule"
+
+    def _get_stock_move_values(
+        self,
+        product_id,
+        product_qty,
+        product_uom,
+        location_id,
+        name,
+        origin,
+        company_id,
+        values,
+    ):
+        """Transfer the SO line description to the move name."""
+        res = super()._get_stock_move_values(
+            product_id,
+            product_qty,
+            product_uom,
+            location_id,
+            name,
+            origin,
+            company_id,
+            values,
+        )
+        if values.get("sale_line_id"):
+            line = self.env["sale.order.line"].browse(values["sale_line_id"])
+            res["name"] = line.name
```

Comment:
IMHO, this does more than it should for a 'report' module.

It should be done in a specific module like 'sale_stock_order_line_name' or ...

Reply:
The feature and the goal is the same than in previous versions, so we don't see the need of adding a new module. It's explained already on the README, so people knows what to get when they install it, and those with the module installed in previous versions will get the same functionality. The question is that this was handled by Odoo itself previously, and now it's not.

Reply:
Yes, I understood that.

But, you mixed two functionalities in the same module. This should be avoided (as some people maybe want to get the transmission of sale line description to move without the custom description module...)

Reply:
They are separated reports, so no problem on that. And they can also create another module with just that feature if they want.

Reply:
> And they can also create another module with just that feature if they want.

So, duplicated code

Reply:
Yes, but that's only if someone like you insists :stuck_out_tongue_closed_eyes:

Reply:
Try to do a better OCA world. I'm a little bit utopian :-)

Reply:
Well, I know that, but sometimes we need to be pragmatic, and I assure you that we already had this discussion before going this way, as the other will require a lot of more work for the migration path of our customers, and we are talking about 10 lines of code (20 with black, hehe). It's a good trade-off.